.github/workflows/performance.yaml

```diff
@@ -0,0 +1,155 @@
+name: Performance Test K8s-snap
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  BASE_SHA: ${{ github.before || github.event.pull_request.base.sha }}
+  BASE_BRANCH: ${{ github.base_ref || github.ref }}
+  TARGET_SHA: ${{ github.sha }}
+
+jobs:
+  build:
+    name: K8s-snap Performance Test Dqlite ${{ matrix.dqlite }}
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        dqlite: ["v1.16.7", "v1.17.0-rc1"]
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@v2
+        with:
+          egress-policy: audit
+      - name: Checking out repo
+        uses: actions/checkout@v4
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.8"
+      - name: Install tox
+        run: |
+          pip install --upgrade pip
+          pip install tox
+      - name: Install tox
+        run: pip install tox
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.21"
+      - name: Install lxd
+        run: |
+          sudo snap refresh lxd --channel 5.21/stable
+          sudo lxd init --auto
+          sudo usermod --append --groups lxd $USER
+          sg lxd -c 'lxc version'
+      - name: Ensure lxd network traffic flows
```

Comment:
Is this the right name for this step?

Reply:
I'll find a better name: docker is dropping some of lxd's network packets which makes the cluster communication impossible.